build.sbt

```diff
@@ -42,24 +42,23 @@ lazy val server = (project in file("server"))
       (baseDirectory.value / "test" / "tests" * "*").get map
         (x => x -> ("test/tests/" + x.getName)),
 
+    resourceDirectory in Test := baseDirectory.value / "test-resources",
```

Comment:
we have scala packages in server/test. Do you propose to add resources as package in test?